bindings/rust/src/lib.rs

```diff
@@ -76,6 +99,24 @@ impl FizzyErrorBox {
                 .into_owned()
         }
     }
+
+    /// Return a translated error object.
+    fn error(&self) -> Option<Error> {
+        match self.0.code {
+            sys::FizzyErrorCode_FIZZY_SUCCESS => None,
+            sys::FizzyErrorCode_FIZZY_ERROR_MALFORMED_MODULE => {
+                Some(Error::MalformedModule(self.message()))
```

Comment:
Could also think getting rid of `code()` and `message()` and only having `error()`.